### ISSUE TITLE
Time for a new version?

### ISSUE THREAD

**Issue by antonsoroko** [elgatito/plugin.video.elementum#1022]
I think it is time for a new version - a lot of changes were made already.
As an intermediate step a beta version can be built and attached here, then after a week - make a proper release.
What do you think?

**Comment by stefoid** [comment]
Hi, great work!

I am just now  upgrading elementum on my Android TV box, installed a long time ago - 1.57   ->  1.99
Unfortunately, it crashes on startup multiple times, then gives up.
I installed from repo, not zip

I may have done stupid things before installing, such as manually uninstalled burst.  I did not also choose "clear settings"
Then I installed 1.99 over the top of existing 1.57

So maybe it is in a confused state bcause of my actions?

Would be nice to have 'nuclear option'  to enforce pure start
